.travis.yml

```diff
@@ -7,10 +7,18 @@ matrix:
     - os: linux
       dist: trusty
       env: PYTHON=2.7 PYTHONWARNINGS=ignore
+      before_install:
+        # `libtcmalloc-minimal4` is for this TensorFlow issue: https://github.com/tensorflow/tensorflow/issues/6968.
+        - sudo apt-get update -qq
+        - sudo apt-get install -y libtcmalloc-minimal4
```

Comment:
I will get something like

```
*** Error in `python -m pytest -svvv python/ray/tune/test/ray_trial_executor_test.py -k test_start_stopu': corrupted size vs. prev_size: 0x000055bd463f1f00 ***
Fatal Python error: Aborted
Stack (most recent call first):
/home/travis/.travis/job_stages: line 104: 17219 Aborted                 (core dumped) python -m pytest -svvv python/ray/tune/test/ray_trial_executor_test.py -k test_start_stopu
The command "python -m pytest -svvv python/ray/tune/test/ray_trial_executor_test.py -k test_start_stopu" exited with 134.
```

when exiting a pytest like `test_start_stopu`.

The test `test_start_stopu` only includes:

```python
def test_start_stopu():
    import ray.rllib.models
    import scipy.signal

    ray.init()
    ray.shutdown()
```

It only happens when:

1. Use trusty linux under travis CI
2. Use both tensorflow and scipy.signal

I have checked my code and profiled with valgrind but I cannot see any memory leaks in my code, so I suppose it could come from tensorflow. Use `libtcmalloc-minimal4` in the tensorflow issue solves this problem.